.travis.yml

```diff
@@ -1,4 +1,39 @@
 language: node_js
+
 node_js:
-  - "6"
-  
+- "8.9.4"
+
+before_install:
+- npm install -g version-from-git
+- version-from-git --travis
+
+install:
+- npm install
+
+before_script:
+- npm run build
+- npm run build-test
+
+script:
+# Our tests currently don't run on Travis, disabling them for now
+- npm --version
+
+before_deploy:
+- git config --local user.name "Bot Framework"
+- git config --local user.email "botframework@microsoft.com"
+
+deploy:
+- provider: npm
+  api_key:
+    secure: "Mgt7i6P52nq77RdBUlcS1EBosxW6Qd6mM86A6RLQKzNJTWMW6GM0YhrqVHT6HQxrqcTRvuSHxJtOKyLoLPou2lWCZbsSy0ANb8tEJcurTF4lLvcTgMN+G5YD74F4RHRyLNW6rHDKvYvBl4iDLE6rW/B6Ew6uORMdlzJOJzjw075/7AoJIjB60f/Hf2aAx3q0wcWxlPyTqh6YD2mJ1GaiflTalxkYalr/Xh3bevrP4I2XyZV1vaCJqDZvrUVlYfSOCJRyBUSrCemkc6nkSTwdbY38L4k7ilrNgm5QfugoaKIPe600WCy4uPp0woz796rdurppsKeLmMSCKKu2mIqz1CT6mf8CR8hQAzGhKqsF0Rydjz6N/OF2XnFbtPZzu/FQJYED9H1UViUAHXsw528Ul9NSseMCVIZirRT+ysIB8Ny9oBddfRyifxnv2T9Zorm7y1V0nrmMXgCrZIvfgvOVXqij7euC4dgnOG4w0eG/Vd1EW87rskpZswV8gx/7ZsVOY/1NFDVxBi//EBSBOLctOSoZF8snT0+Duvw5Id3pXKvqm+ffI19xBe8Bua64fMg7dHWqZN/ixjTqeySBAYDPtRx/0wBSBwMIDh/Syux+fgjXpqf87HJc3vbjAXyuwPyO9mlblpm02JrZmG8Yik1krVvnLWzvqGqrmVKsLKmndn8="
```

Comment:
Is this a secret?

Reply:
This is an encrypted secret, and is only decryptable by Travis CI. It is designed to be safe to expose publicly without worrying being hacked.